book-examples/leptos/src/default/card/card.rs

```diff
@@ -0,0 +1,87 @@
+use leptos::prelude::*;
+use lucide_leptos::{BellRing, Check};
+use shadcn_ui_leptos_button::default::Button;
+use shadcn_ui_leptos_card::default::{
+    CardContent, CardDescription, CardFooter, CardHeader, CardTitle,
+};
```

Comment:
These imports should be `use crate::default::components::ui::{}`, just like `Card` below.